src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/ciag/UpdateCiagInductionRequestMapper.kt

```diff
@@ -1,17 +1,17 @@
-package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.resource.mapper.induction
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.resource.mapper.induction.ciag
 
 import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.dto.UpdateInductionDto
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.UpdateCiagInductionRequest
 
 @Component
 class UpdateCiagInductionRequestMapper(
-  private val workOnReleaseMapper: WorkOnReleaseResourceMapper,
+  private val workOnReleaseMapper: CiagWorkOnReleaseResourceMapper,
   private val qualificationsAndTrainingMapper: QualificationsAndTrainingResourceMapper,
```

Comment:
I didn't rename this one as the name would be very grande (and I'm hoping this code doesn't need to stay around for too long anyway! :))